.github/stale.yml

```diff
@@ -0,0 +1,24 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
That's a bit too short for my taste. I would prefer if this is e.g. 30 days so people have time to react if coming back from vacation and etc. @mickaelistria wdyt?